src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big_form"
+      action="/"
+      method="post"
+      data-qa="big">
+      <img
+        class="big_icon"
+        src="images/Search.svg"
+        alt="Search_icon">
```

Comment:
Fix your code style everywhere please
```suggestion
      <img
        class="big_icon"
        src="images/Search.svg"
        alt="Search_icon"
      >
```